cdk/data.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Add note that these configurations are for test environment and not for production use

Reply:
Added in latest rev